app/react-library/query-options.mdx

```diff
@@ -0,0 +1,13 @@
+#### options
+
+TanStack Query parameters. See the
+[TanStack Query query](https://tanstack.com/query/v5/docs/framework/react/reference/useQuery)
+docs for more info.
+
+<div className="theme-surface p-4 rounded-md mt-4">
+  Metrom react library does not support all TanStack Query parameters, like
```

Comment:
React should be written as "React"

Reply:
Fixed in e642e989741b8747d30e48a923c20e23c9263221.